src/lib/DiscordWebhook/utils.ts

```diff
@@ -5,7 +5,7 @@ import Bot from '../../classes/Bot';
 
 import { Webhook } from './interfaces';
 
-export function getPartnerDetails(offer: TradeOffer, bot: Bot, callback: (err: any, details: any) => void): any {
+export function getPartnerDetails(offer: TradeOffer, bot: Bot, callback: (err: Error, details: any) => void): any {
```

Comment:
I don't think in TS you can type an error, it's just a blob of Error. We can disect them but TS recommends not throwing errors unless you are ready to catch it and just dump the JSON.stringify the error to a log meesage etc.

I understand this is a function call but the tension between error handling in TS vs JS causes eslint to go crazy.